src/components/home/experts-feed/experts-carousel.tsx

```diff
@@ -0,0 +1,121 @@
+"use client";
+
+import { Link } from "@/navigation";
+import Image from "next/image";
+import { useTranslations } from "next-intl";
+
+import Autoplay from "embla-carousel-autoplay";
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+} from "@/components/ui/carousel";
+import { cn } from "@/lib/utils";
+import { Button } from "../../custom/button";
+import { DUMMY_USERS } from "@/lib/constants";
+
+export const ExpertsCarousel = () => {
+  const t = useTranslations("Home.expertsFeed.carousel");
+
+  return (
+    <Carousel
+      plugins={[
```

Comment:
Seems like plugins array can be declared outside of component function scope. There is no need in creating new array on each component render